airesources/OCaml/game.ml

```diff
@@ -0,0 +1,256 @@
+(* OCaml Starter for Halite II on Halite.io
+   This code is public domain. There is no warranty.
+ *)
+
+open Td;;
+
+let set_planets_docked_ships state =
+  List.iter (fun planet ->
+    planet.docked_ships <- List.filter (fun ship ->
+      ship.status = `Docked
+      && ship.docked_planet_id = planet.p_entity.id
+    ) state.all_ships
+  ) state.planets
+;;
+
+let update_map state =
+  state.turn <- state.turn + 1;
+  state.last_update <- Unix.gettimeofday ();
+  Debug.info (Printf.sprintf "--- Begin Turn %i ---\n" state.turn);
+  Networking.parse_map state (Networking.get_string());
+  set_planets_docked_ships state;
+;;
+
+let init state name initial_bot_setup =
+  state.my_id <- int_of_string (Networking.get_string ());
+  Debug.setup_logging state.my_id name;
+  Networking.read_width_height state (Networking.get_string ());
+  update_map state;
+  initial_bot_setup state;
+  Networking.send_string name;
```

Comment:
I'm still a bit confused about this. This line sends bot name to the server right?